src/js/view/components/nextuptooltip.js

```diff
@@ -157,6 +158,7 @@ define([
             var streamType = this._model.get('streamType');
             if (streamType === 'LIVE' || streamType === 'DVR') {
                 model.off('change:duration', this.onDuration, this);
+                model.off('change:position', this.onElapsed, this);
```

Comment:
Live/DVR/VOD state can change when duration changes. Maybe these listeners should be removed and added on 'change:streamType'?